build.sh

```diff
@@ -125,7 +125,7 @@ if compare_versions $(vagrant plugin list | grep 'vagrant-vmware' | cut -d' ' -f
   providers="vmware $providers"
 fi
 
-if compare_versions $(vagrant plugin list | grep 'vagrant-reload' | cut -d' ' -f2 | tr -d '(' | tr -d ')') $min_vagrantreload_ver false; then
+if compare_versions $(vagrant plugin list | grep 'vagrant-reload' | cut -d' ' -f2 | tr -d '(' | tr -d ')' | tr -d ',') $min_vagrantreload_ver false; then
```

Comment:
This was offered as a suggestion for a separate issue, did you test this change on other platforms?

Reply:
I have not been able to test it too much on other Linux distros as I have been having a hard time getting vm's to build and run on my Linux vm's. I did see this in a previous post and I incorporated it.